web/src/test/java/se/inera/intyg/rehabstod/web/controller/api/SickLeaveControllerTest.java

```diff
@@ -71,11 +74,11 @@ void shouldCallGetActiveSickLeavesService() {
                 TEXT_SEARCH,
                 UNANSWERED_COMMUNICATION
             );
-        when(getActiveSickLeavesResponseService.get(any(), anyBoolean())).thenReturn(
+        when(getActiveSickLeavesResponseService.get(any(), anyBoolean(), false)).thenReturn(
```

Comment:
This comment is not specific to this line...

Include a test that verifies explicitly that PDL logging should be sent.